spec/factories/user_job.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :user_job do
+    association :good_job, factory: :good_job
+    user_id { SecureRandom.uuid } # Assuming user_id is a UUID
```

Comment:
Just want to check this assumption that user_id is a UUID still stands? Can this comment be removed?